fastai/tabular/learner.py

```diff
@@ -21,7 +25,15 @@ def predict(self, row):
 
 # Cell
 @delegates(Learner.__init__)
-def tabular_learner(dls, layers=None, emb_szs=None, config=None, n_out=None, y_range=None, **kwargs):
+def tabular_learner(
+        dls:TabularDataLoaders, # Set of tabular dataLoaders
+        layers:list=None, # Sequence of neurons per linear layer
+        emb_szs:list=None, # Tuples of n_unique by embedding_size for all categorical features
+        config:dict=None, # Config params for TabularModel
+        n_out:int=None, # Number of output/classes
+        y_range:(float, float)=None, # Low and high for the sigmoid function
+        **kwargs
+    ):
```

Comment:
```suggestion
def tabular_learner(
        dls:TabularDataLoaders,
        layers:list=None, # Size of the layers generated by `LinBnDrop`
        emb_szs:list=None, # Tuples of `n_unique, embedding_size` for all categorical features
        config:dict=None, # Config params for TabularModel from `tabular_config`
        n_out:int=None, # Final output size of the model
        y_range:(float, float)=None, # Low and high for the final sigmoid function
        **kwargs
    ):
```

Reply:
thanks a lot for the suggestions/feedback, @muellerzr. Sure will make the necessary changes.